conf/log4j.properties

```diff
@@ -12,63 +12,121 @@
+appender.A2.policies.type=Policies
+appender.A2.policies.size.type=SizeBasedTriggeringPolicy
+appender.A2.policies.size.size=100KB
+appender.A2.layout.type=PatternLayout
+appender.A2.layout.pattern=%-4r [%t] %-5p %c %x - %m%n
 
-# A2 uses PatternLayout.
-log4j.appender.A2.layout=org.apache.log4j.PatternLayout
-log4j.appender.A2.layout.ConversionPattern=%-4r [%t] %-5p %c %x - %m%n
-
-# Test Case Step tracer
-#log4j.logger.testcase.trace=TRACE, A3
-#log4j.logger.com.zimbra.qa.selenium.framework.util.HarnessException=A3
+logger.A2.name = A2File
+logger.A2.level = info
+logger.A2.additivity = false
+logger.A2.appenderRef.file.ref = fileAppender
 
 # A2 is set to be a FileAppender.
-log4j.appender.A3=org.apache.log4j.FileAppender
-log4j.appender.A3.File=${outputDirectory}/testcase.txt
-log4j.appender.A3.MaxFileSize=100KB
-log4j.appender.A3.Append=false
+appender.A3.type=org.apache.log4j.RollingFileAppender
+appender.A3.fileName=${outputDirectory}/testcase.txt
+appender.A3.name=a3FileAppender
+appender.A3.policies.type=Policies
+appender.A3.policies.size.type=SizeBasedTriggeringPolicy
+appender.A3.policies.size.size=100KB
+appender.A3.append=false
 
 # A2 uses PatternLayout.
-log4j.appender.A3.layout=org.apache.log4j.PatternLayout
-log4j.appender.A3.layout.ConversionPattern=%m%n
-
-log4j.appender.general=org.apache.log4j.FileAppender
-log4j.appender.general.File=${outputDirectory}/debug.txt
-log4j.appender.general.Threshold=WARN
-log4j.appender.general.append=false
-log4j.appender.general.layout=org.apache.log4j.PatternLayout
-log4j.appender.general.layout.ConversionPattern=%-4r %-5p %c %x - %m%n
-
-log4j.logger.com.zimbra.qa.selenium.framework.util.staf.StafAbstract=INFO,coverage
-log4j.logger.com.zimbra.qa.selenium.framework.util.CodeCoverage=INFO,coverage
-log4j.logger.com.zimbra.qa.selenium.framework.core.ExecuteHarnessMain.MethodListener=INFO,coverage
-log4j.appender.coverage=org.apache.log4j.FileAppender
-log4j.appender.coverage.File=${outputDirectory}/coverage.txt
-log4j.appender.coverage.append=false
-log4j.appender.coverage.layout=org.apache.log4j.PatternLayout
-log4j.appender.coverage.layout.ConversionPattern=%-4r %-5p %c %x - %m%n
-
-log4j.logger.com.zimbra.qa.selenium.framework.util.SleepMetrics=INFO,sleep
-log4j.appender.sleep=org.apache.log4j.FileAppender
-log4j.appender.sleep.File=${outputDirectory}/sleep.txt
-log4j.appender.sleep.append=false
-log4j.appender.sleep.layout=org.apache.log4j.PatternLayout
-log4j.appender.sleep.layout.ConversionPattern=%-4r - %m%n
+appender.A3.layout.type=PatternLayout
+appender.A3.layout.pattern=m%n
+
+logger.A3.name = A3File
+logger.A3.level = info
+logger.A3.additivity = false
+logger.A3.appenderRef.stdout.ref = a3FileAppender
+
+appender.general.type=org.apache.log4j.RollingFileAppender
+appender.general.fileName=${outputDirectory}/debug.txt
+appender.general.name=generalFileAppender
+appender.general.level=WARN
+appender.general.append=false
+appender.general.layout=org.apache.log4j.PatternLayout
```

Comment:
Use PatternLayout only as org.apache.log4j.PatternLayout is belongs to log4j-1.2.16, Please correct for other places too.